tests/datadriven/test_variants.py

```diff
@@ -428,5 +429,6 @@ def _hashVariant(self, record):
         if record.ALT[0] is None:
             alts = tuple()
         else:
-            alts = tuple([str(substitution) for substitution in record.ALT])
-        return hashlib.md5(record.REF + str(alts)).hexdigest()
+            alts = tuple([unicode(sub) for sub in record.ALT])
```

Comment:
Why are we changing this? `unicode` is Python 2 only, so please leave as `str` unless this breaks something.